### ISSUE TITLE
only show the name of the virtual environment

### ISSUE THREAD

**Issue by OdinTech3** [fisherman/mono#1]
When you activate a virtual environment, it shows the full path to it rather than only showing the name of the virtual env folder. Is it possible to only show the name of folder the virtual environment is located in?

**Comment by JorgeBucaran** [comment]
@OdinTech3 I don't remember doing anything so that it would show the name of the virtual env folder. Perhaps the full path to the venv folder is simply picked up as the name of the directory by `pwd`. I have no idea! 😄🤔

**Comment by OdinTech3** [comment]
Yeah I think so. Instead of using `pwd` and obtaining the full path, you could use `basename "$VIRTUAL_ENV"` which should just return the folder name of the virtual env if it activated.

**Comment by JorgeBucaran** [comment]
@OdinTech3 Sounds good! Go ahead and send me a PR if you want this and I'll merge it. 👋

**Comment by JorgeBucaran** [comment]
Just invited you to admin this repo! https://github.com/fisherman/mono/invitations
